src/SprykerEco/Zed/Computop/Business/Processor/NotificationProcessor.php

```diff
@@ -60,14 +72,66 @@ protected function executeSaveComputopNotificationTransaction(
         ComputopNotificationTransfer $computopNotificationTransfer
     ): ComputopNotificationTransfer {
         $this->computopEntityManager->savePaymentComputopNotification($computopNotificationTransfer);
-        $isProcessed = $this->computopEntityManager->updatePaymentComputopOrderItemPaymentConfirmation(
-            $computopNotificationTransfer,
-            $this->getCurrentOrderItemEntityStatus($computopNotificationTransfer),
+        $paymentComputopTransfer = $this->computopRepository->findComputopPaymentByComputopTransId(
+            $computopNotificationTransfer->getTransIdOrFail(),
         );
 
-        $computopNotificationTransfer->setIsProcessed($isProcessed);
+        if (!$paymentComputopTransfer) {
+            return $computopNotificationTransfer->setIsProcessed(false);
+        }
+
+        $this->updateComputopPayment($paymentComputopTransfer, $computopNotificationTransfer);
+
+        $isProcessed = $this->updatePaymentComputopOrderItems($paymentComputopTransfer, $computopNotificationTransfer);
+
+        return $computopNotificationTransfer->setIsProcessed($isProcessed);
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\ComputopPaymentComputopTransfer $computopPaymentComputopTransfer
+     * @param \Generated\Shared\Transfer\ComputopNotificationTransfer $computopNotificationTransfer
+     *
+     * @return void
+     */
+    protected function updateComputopPayment(
```

Comment:
this method is useless
please put it's code directly to a place where it used

Reply:
Fixed